client/src/lib/utils/setters.ts

```diff
@@ -26,9 +26,15 @@ export const setVisibility = ({ v = true, type = 'mapbox' }: SetVisibilityProps)
   return v;
 };
 
+type SetFrameProps = { frame: number };
+export const setFrame = ({ frame = 0 }: SetFrameProps) => {
```

Comment:
Do we need a function to override a simple a number? I think you can achieve it without a function

Reply:
I mean, JSONConverter can handle numbers, the function is because you need to make any operation before returning the number. Otherwise is adding an unnecessary dependency